docker/README.md

```diff
@@ -90,7 +90,7 @@ Connect `Bob` node to `Alice` node.
 
 ```bash
 # Run "Bob" node and log into it:
-$ docker-compose run -d --name bob lnd_btc
+$ docker-compose run -d bob
```

Comment:
This still needs to inlcude the `--name` parameter, since if not the container will be given a random name.